libs/events/import/api/src/lib/import/events-import.model.ts

```diff
@@ -0,0 +1,124 @@
+import {Field, ObjectType, InputType, registerEnumType, Int} from '@nestjs/graphql'
+import GraphQLJSON from 'graphql-type-json'
+
+export enum Providers {
```

Comment:
The problem with this is that all providers have to come by WePublish and can't be implemented by operators. Which imo defeats the purpose of it.